edx-platform/pearson-theme/lms/templates/header/navbar-authenticated.html

```diff
@@ -14,6 +14,7 @@
   show_explore_courses = settings.FEATURES.get('COURSES_ARE_BROWSABLE')
   show_sysadmin_dashboard = settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff
   self.real_user = getattr(user, 'real_user', user)
+  show_catalogue_link = configuration_helpers.get_value('show_catalogue_link', 'true')
```

Comment:
I think the default value should be: show_explore_courses, just to keep the edx-platform logic in place. @amalbas

Reply:
@Squirrel18 should the variable in the site configuration be show_explore_courses too?

Reply:
show_explore_courses is already defined on 14 line, and takes its value from the platform settings. @amalbas

Reply:
So should this pr be closed? there is no extra changes if we use the show explore courses variables.

Reply:
```suggestion
  show_catalogue_link = configuration_helpers.get_value('show_catalogue_link', show_explore_courses)
```
Sorry I think I explained it really badly, this change is to use a site configuration instead of the platform setting, but if the site setting is not set then the default should be the platform setting.